packages/integration-sdk-runtime/src/api/index.ts

```diff
@@ -41,9 +42,19 @@ export function createApiClient({
     headers.Authorization = `Bearer ${accessToken}`;
   }
 
+  const maxTimeout = process.env.SDK_API_CLIENT_MAX_TIMEOUT
```

Comment:
nit: Consider pulling out into a separate method. Maybe `determineMaxTimeout`?

Reply:
I think we should avoid environment variables when we can. Environment variables are global state and require some clever testing strategies.

Instead of using an environment variable, I think `createApiClient` should accept a `retryConfig` argument. `createApiClient` is called from the managed integration SDK. I think we could allow passing configuration along to the managed SDK from the integration itself and have the managed SDK pass along the configuration to `createApiClient`.

```ts
createApiClient({
  ...,
  retryConfig: {
    maxTimeout: 10000
  }
});
```

If `retryConfig` is not supplied, then the value will be `undefined` and `Alpha` would just use the defaults. Once we are happy with the `maxTimeout` that we assign from the managed integration side, we should make a follow-up change to the SDK to use that value as the default.

Thoughts? @VDubber @zemberdotnet

Reply:
Agreed (see [here](https://github.com/JupiterOne/sdk/pull/653#issuecomment-1070986891)). Also, EnvVars don't provide an obvious interface.

Reply:
In the latest PR the `createApiClient` function now looks like:
```ts
export function createApiClient({
  apiBaseUrl,
  account,
  accessToken,
  retryOptions,
}: CreateApiClientInput): ApiClient {
```
The `retryOptions` is optional. I went with `retryOptions` as that is what it's called in `Alpha`. 

Alternatively, we could pull in the full `AlphaOptions` from [Alpha](https://github.com/lifeomic/alpha/blob/9adf37ed32a030990e564d41cd09607fc7e6c9aa/src/Alpha.d.ts#L12). That would make `createApiClient` look more like:
```ts
export function createApiClient({
  account,
  accessToken,
  alphaOptions,
}): CreateApiClientInput): ApiClient {
```

`AlphaOptions` extends `AxiosOptions` so `apiBaseUrl` would be included on the `AlphaOptions` object. This is probably a bigger change than necessary and might break something - I'm not sure. I wanted to put it out there incase anyone had thoughts, but I'm happy with the existing change as well.